ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -36,24 +77,9 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
       reload: true,
     });
 
-    if (target.host_sources) {
-      const hostSets = await Promise.all(
```

Comment:
It's fine that you split this out into functions but to be honest, all that was needed was a try catch around these `Promise.all` (or the entire thing) and it would've just returned an empty array of hosts